### ISSUE TITLE
Stock list (feature request)

### ISSUE THREAD

**Issue by TheBrokenWasp** [Qluxzz/avanza#20]
So I haven't actually used this yet but I went through the documentation and I see that it does not seem to have a function to get s list of available stocks.

I want to be able to detect whenever a new stock is released to the market (for a stupid experiment of mine which will most likely cost me dearly lol) but I don't see that option anywhere...

Would it be possible to add a function to get all the stocks on the market? At least this way I can just make something that checks every once in a while if there is something new.

**Comment by Qluxzz** [comment]
Only if similar functionality already exists on the website or app today.

This library's purpose is to only expose the existing functionality, not create anything new.

**Comment by TheBrokenWasp** [comment]
After a quick search I found this: https://www.avanza.se/aktier/lista.html

I am assuming this is the list of things you can buy on avanza.

**Comment by robiwano** [comment]
https://andreandersson.se/avanza/avanza.html#avanza.avanza.Avanza.get_instrument ??

**Comment by robiwano** [comment]
I use that in conjunction with get_certificate_info to get hold of specific certificates.

**Comment by JonasOlson** [comment]
@robiwano: Right, but that means getting information about a single instrument, with a known ID, rather than getting a list of all instruments, which is what is requested here. Doesn't it?

**Comment by faddi** [comment]
I use the following to extract all ids from https://www.avanza.se/aktier/lista.html:
`curl https://www.avanza.se/frontend/template.html/marketing/advanced-filter/advanced-filter-template\?1623583358324\&parameters.startIndex\=0\&parameters.maxResults\=100000\&parameters.selectedFields%5B0%5D\=LIST | grep "kop/[0-9]*" | grep -oh "[0-9]*" | sort | uniq > all.txt`

**Comment by JonasOlson** [comment]
@faddi: Thanks! I was not aware of that URL. What else can you do with it? Is there a way to list instruments other than stocks, or find out what other parameters are available?

**Comment by faddi** [comment]
@JonasOlson I found it by inspecting calls going out while doing filter changes at https://www.avanza.se/aktier/lista.html. You'll have to check the calls made on the other instrument listing pages.

**Comment by dejmail** [comment]
> I use the following to extract all ids from https://www.avanza.se/aktier/lista.html: `curl https://www.avanza.se/frontend/template.html/marketing/advanced-filter/advanced-filter-template\?1623583358324\&parameters.startIndex\=0\&parameters.maxResults\=100000\&parameters.selectedFields%5B0%5D\=LIST | grep "kop/[0-9]*" | grep -oh "[0-9]*" | sort | uniq > all.txt`
> 
> Gives about 10k ids right now

@faddi  Is this still working for you ? I can't seem to request more than 500 ids without getting a 404.
